04-vector-raster-integration-advanced.Rmd

```diff
@@ -277,6 +293,13 @@ plot(plot.locationSp_HARV,
 
 ```
 
```

Comment:
I feel that some explanation is needed.  The challenge says to crop the raster to the vector, so one would expect that the resulting plot would have all of the plot locations on the (visible) raster CHM layer.  But one isn't, why?  Understanding why isn't necessarily intuitive as we just cropped the raster to the vector extent so it should be the same... but as the raster already has a smaller geographic extent on the western edge the vector and raster layers don't have the same extent after the cropping.   

Does this work better: 

In the plot above, created in the challenge, all the vegetation plot locations
(blue) appear on the Canopy Height Model raster layer except for one. One is
situated on the white space. Why? 

Look back at the first figure in this lesson showing the relative extents of
all the spatial objects.  Notice that the veg. plot extent (black) extends
slightly farther west than the CHM raster (green) extent. We cropped the raster
to the vector extent, but there isn't any raster data in that western region of
our plot area so the vegetation plot in that area appears outside the extent of
the raster data.